pyproject.toml

```diff
@@ -135,6 +135,41 @@ source = "https://github.com/scipy/scipy"
 download = "https://github.com/scipy/scipy/releases"
 tracker = "https://github.com/scipy/scipy/issues"
 
-
 [tool.doit]
 dodoFile = "dev.py"
+
+
+[tool.cibuildwheel]
+skip = "cp36-* cp37-* pp* *_ppc64le *_i686 *_s390x *-musllinux*"
+build-verbosity = "3"
+before-test = "pip install -r {project}/tools/wheels/test_requirements.txt"
+test-command = "bash {project}/tools/wheels/cibw_test_command.sh {project}"
+
+[tool.cibuildwheel.linux]
+manylinux-x86_64-image = "manylinux2014"
+manylinux-aarch64-image = "manylinux2014"
+before-build = "bash {project}/tools/wheels/cibw_before_build_linux.sh {project}"
+test-skip = "*_aarch64"
+
+[tool.cibuildwheel.macos]
+# If universal2 wheels are wanted use delocate-fuse
+# TODO figure out arm64 cross compilation
+before-build = "bash {project}/tools/wheels/cibw_before_build_macos.sh {project}"
+test-skip = "*_arm64 *_universal2:arm64"
+
+[tool.cibuildwheel.windows]
+before-build = "bash {project}/tools/wheels/cibw_before_build_win.sh {project}"
+repair-wheel-command = "bash ./tools/wheels/repair_windows.sh {wheel} {dest_dir}"
+
+[[tool.cibuildwheel.overrides]]
+select = "*-win32"
+# don't use double backslash for path separators, they don't get passed
+# to the build correctly
+# use cmake detection for openBLAS because the built-in pkg-config is defective
+environment = { CXXFLAGS="-m32", CFLAGS="-m32", LDFLAGS="-m32", CMAKE_PREFIX_PATH="c:/opt/openblas/if_32/32" }
```

Comment:
It looks like you tested this, but I don't quite understand how. It looks to me like this needs commenting out because there is no 32-bit Windows build.